linux/install_centos7_nginx.sh

```diff
@@ -6,9 +6,10 @@ WEBSESSION=${WEBSESSION:-false}
 OMEROVER=${OMEROVER:-latest}
 PGVER=${PGVER:-pg10}
 ICEVER=${ICEVER:-ice36}
+VIRTUALENV=${VIRTUALENV:-/home/omero/omeroenv}
 
-source `dirname $0`/settings.env
-source `dirname $0`/settings-web.env
+. `dirname $0`/settings.env
```

Comment:
Why the change from `source` to `.`?

Reply:
i had problem on debian

Reply:
Understood. i.e. https://wiki.ubuntu.com/DashAsBinSh#source